### ISSUE TITLE
Tabs+spaces in performance statistics output make copying wonky

### ISSUE THREAD

**Issue by justincwatt** [JoeDog/siege#126]
This output in the terminal (Apple Terminal v2.7.3) using Siege v3.0.5:

![screen shot 2018-04-12 at 11 54 36 am](https://user-images.githubusercontent.com/1240939/38697971-54a5fa86-3e48-11e8-82bc-9b5ccfd142b9.png)

Looks like this when I copy and paste elsewhere (in Slack for instance):

![screen shot 2018-04-12 at 11 57 44 am](https://user-images.githubusercontent.com/1240939/38698111-caeea454-3e48-11e8-900f-a87e45a2311b.png)

Would be nice to align just with spaces.

**Comment by JoeDog** [comment]
Good point.  I should have used print formatting.  I'll try to change that
soon.
